API/controller/mealController.js

```diff
@@ -70,11 +70,26 @@ const updateAmeal = (req, res) => {
   });
 };
 
+// TO REMOVE A MEAL OPTION
+const deleteAmeal = (req, res) => {
+  const meal = dummydb.meals.find(data => data.id === parseInt(req.params.id));
```

Comment:
You want to use `Number` here instead of `parseInt`. Number converts the entire string into a number, while parseInt parses up to the first non-digit.